plugins/amt/fu-plugin-amt.c

```diff
@@ -544,15 +553,35 @@ void
 fu_plugin_init (FuPlugin *plugin)
 {
 	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+	fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
 }
 
 gboolean
 fu_plugin_coldplug (FuPlugin *plugin, GError **error)
 {
 	g_autoptr(FuDevice) dev = NULL;
-	dev = fu_plugin_amt_create_device (error);
+	dev = fu_plugin_amt_create_device (plugin, error);
 	if (dev == NULL)
 		return FALSE;
 	fu_plugin_device_add (plugin, dev);
 	return TRUE;
 }
+
+void
+fu_plugin_add_security_attrs (FuPlugin *plugin, FuSecurityAttrs *attrs)
+{
+	FuPluginData *data = fu_plugin_get_data (plugin);
+	g_autoptr(FwupdSecurityAttr) attr = NULL;
+
+	if (!fu_common_is_cpu_intel ())
+		return;
+
+	attr = fwupd_security_attr_new ("com.intel.AMT");
+	fwupd_security_attr_set_level (attr, FWUPD_SECURITY_ATTR_LEVEL_SYSTEM_PROTECTION);
+	fwupd_security_attr_set_name (attr, "Intel AMT");
+	fu_security_attrs_append (attrs, attr);
+	if (data->provisioned)
+		fwupd_security_attr_set_result (attr, "provisioned");
```

Comment:
I think the result needs to be sentance case, e.g. `Provisioned`

Reply:
fixed